lib/wsdl.js

```diff
@@ -1291,9 +1291,20 @@ WSDL.prototype.xmlToObject = function(xml) {
 
     if(hasNonXmlnsAttribute)obj[self.options.attributesKey] = elementAttributes;
 
+    // Pick up the schema for the type specified in element's xsi:type attribute.
+    var _xsiTypeSchema;
+    var _xsiType = elementAttributes['xsi:type'];
+    if (_xsiType) {
+      var _type = splitNSName(_xsiType);
+      var _typeDef = self.findParameterObject(xmlns[_type.namespace], _type.name);
+      if (_typeDef) {
+        _xsiTypeSchema = _typeDef.description(self.definitions);
+      }
+    }
+
```

Comment:
Would you mind renaming your variables and remove the trailing underscore, for example `_xsiType` --> `xsiType`?  I'm aware that this is a bit picky, but in general underscores should be "reserved" for private members and methods :+1: